tv_apps/dashboard/index.html

```diff
@@ -33,7 +34,26 @@
     <div id="background" class="background"></div>
     <div id="background-mask" class="background-mask"></div>
     <div role="region" id="main-section" tabIndex="-1">
-      <section id="center-panel"></section>
+      <section id="center-panel">
+        <div id="digital-clock">
+          <p id="time-now" class='time'>
+            <span id="hours-tens-digit" class="hours-tens-digit"></span>
+            <span id="hours-units-digit" class="hours-units-digit"></span>
+            <span class="colons">:</span>
+            <span id="minutes-tens-digit" class="minutes-tens-digit"></span>
+            <span id="minutes-units-digit" class="minutes-units-digit"></span>
+            <span class="colons">:</span>
+            <span id="seconds-tens-digit" class="seconds-tens-digit"></span>
+            <span id="seconds-units-digit" class="seconds-units-digit"></span>
+          </p>
+          <p id="date-now" class='date'>
+            <span data-l10n-id="today-is">Today is</span>
```

Comment:
But I'm not quite sure if there are good solutions for putting it as one l10n key since this causes nested l10n query. Let's do some investigation.